### PR TITLE
Correct coordinates, dropdown arrow, dropdown text

### PR DESCRIPTION
**Pull request by AdrielStammler**
Made it so that when the field image is flipped the coordinates are too. Made a custom dropdown arrow for the dropdowns.

### CODE REVIEW

File: app/src/main/java/com/cpr3663/cpr_scouting_app/Match.java
Comment (Jeff-Stammler) [inline app/src/main/java/com/cpr3663/cpr_scouting_app/Match.java:1]: Remove updating test status with coordinates

File: app/src/main/res/layout/post_match.xml [hunk 1]
```diff
@@ -6,15 +6,15 @@
...
-    tools:context=".PostMatch">
+    tools:context=".PostMatch"
+    tools:ignore="VisualLintBounds">
 
     <LinearLayout
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         android:orientation="vertical"
-        app:layout_constraintEnd_toEndOf="parent"
-        app:layout_constraintStart_toStartOf="parent"
```
Comment (Jeff-Stammler) [inline app/src/main/res/layout/post_match.xml:16]: What changed here and why?

File: gradle/libs.versions.toml
Comment (Jeff-Stammler) [inline gradle/libs.versions.toml:1]: You should just omit these from your checking and leave it in your stash

File: gradle/wrapper/gradle-wrapper.properties
Comment (Jeff-Stammler) [inline gradle/wrapper/gradle-wrapper.properties:1]: You should just omit these from your checking and leave it in your stash

File: settings.gradle.kts
Comment (Jeff-Stammler) [inline settings.gradle.kts:1]: You should just omit these from your checking and leave it in your stash

**Review by Jeff-Stammler — CHANGES_REQUESTED** [review]

File: app/src/main/res/layout/post_match.xml (hunk 1 above)
Comment (AdrielStammler) [reply app/src/main/res/layout/post_match.xml:16]: I don't think I changed anything, it must have been an accident. Do you want it changed or is it fine?

File: gradle/libs.versions.toml
Comment (AdrielStammler) [reply gradle/libs.versions.toml:1]: Ok, I wasn't sure what it was, so I wasn't sure if it was one to omit.

File: gradle/wrapper/gradle-wrapper.properties
Comment (AdrielStammler) [reply gradle/wrapper/gradle-wrapper.properties:1]: Ok, I wasn't sure what it was, so I wasn't sure if it was one to omit.

File: settings.gradle.kts
Comment (AdrielStammler) [reply settings.gradle.kts:1]: Ok, I wasn't sure what it was, so I wasn't sure if it was one to omit.

**Review by Jeff-Stammler — APPROVED** [review]

**Pull request merged**